src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -65,5 +66,92 @@ public static void BuildChainExtraStoreUntrustedRoot()
                 Assert.Contains(chain.ChainStatus, s => s.Status == X509ChainStatusFlags.UntrustedRoot);
             }
         }
+
+        public static IEnumerable<object[]> VerifyExpressionData()
+        {
+            // The test will be using the chain for TestData.MicrosoftDotComSslCertBytes
+            // The leaf cert (microsoft.com) is valid from 2014-10-15 00:00:00Z to 2016-10-15 23:59:59Z
```

Comment:
Yeah, Intermediate is 2013-10-30 to 2023-10-30.  Root is an optimistic 2006-11-07 to 2036-07-16.